exercises/sum-of-multiples/canonical-data.json

```diff
@@ -1,22 +1,29 @@
 {
   "exercise": "sum-of-multiples",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "cases": [
     {
-      "description": "multiples of 3 or 5 up to 1",
+      "description": "multiples of 3 up to 1",
       "property": "sum",
-      "factors": [3, 5],
+      "factors": [3],
       "limit": 1,
       "expected": 0
     },
     {
-      "description": "multiples of 3 or 5 up to 4",
```

Comment:
I like the changes you're proposing here, adding the single number tests first.

But I think this test is still relevant, with one number below the limit and one number above.

Reply:
I personally just don't think it brings much value when the solution to the test is just going to be
```code
if(limit > 3} return 3;
```
Until you knock out the next couple of tests which force you to refactor, and replace 3 with facotrs[].

I'm not necessarily sold on it, but just my opinion.

Reply:
Maybe not the way that you are implementing the solution, but there are many different approaches. In some of these this will be an issue.